src/modules/shared/components/Navigation/NavigationSection/NavigationSection.tsx

```diff
@@ -0,0 +1,103 @@
+import { Button } from '@meemoo/react-components';
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+
+import { Icon, Overlay } from '@shared/components';
+
+import styles from '../Navigation.module.scss';
+import { NavigationDropdown } from '../NavigationDropdown';
+import { NavigationList } from '../NavigationList';
+
+import { NavigationSectionProps } from './NavigationSection.types';
+
+const NavigationSection: FC<NavigationSectionProps> = ({
+	children,
+	items,
+	placement,
+	renderHamburger,
+	hamburgerProps,
+}) => {
+	// Needed for overlay state. Dropdown state is saved in NavigationDropdown component
```

Comment:
Ik ben dit in vorige pr vergeten wegdoen. Dit is de single-source-of-truth voor de hamburger.

Reply:
De comment zelf? Die mag gerust blijven staan hoor

Reply:
Men punt is dat die fout is. Er wordt geen state bijgehouden in `navigationDropdown`

Reply:
Ah ok 😄 ik zal dat fixen in volgende PR 👍